### DISCUSSION TITLE
Is there  "Classic WebJar" for Tailwind css

### DISCUSSION THREAD

**Discussion by KoHeinCs** [tailwindlabs/tailwindcss#2853] [Help]
I want to use tailwind css in my spring boot project .So I wanna to use Tailwind css as a depedency in my pom.xml file . For example 
, below jquery webjar depedency which used in my pom file .
`<dependency>`
`<groupId>org.webjars</groupId>`
`<artifactId>jquery</artifactId>`
`<version>${jquery.version}</version>`
`</dependency>`

**Comment by CraftingGamerTom** [comment]
Are you talking about [org.webjars.npm » tailwindcss](https://mvnrepository.com/artifact/org.webjars.npm/tailwindcss)?

For example
```
<!-- https://mvnrepository.com/artifact/org.webjars.npm/tailwindcss -->
<dependency>
    <groupId>org.webjars.npm</groupId>
    <artifactId>tailwindcss</artifactId>
    <version>3.2.1</version>
</dependency>

```

**Reply by psoares** [reply]
It looks like it's broken. It depends on packages that don't have webjars available.
For me, when I try to use tailwindcss 3.4.0, the error is: No versions available for org.webjars.npm:isaacs__cliui:jar:[8.0.2,9) within specified range.

**Reply by CraftingGamerTom** [reply]
I did some research on this and see that there was an issue with [isaacs cliui here](https://github.com/isaacs/jackspeak/issues/5)

isaacs_cliui seems to be a dependency of a dependency. There is not much you can do there besides wait for, ask for, or code a fix to whichever one of tailwind's dependencies is asking for that webjar

Your best bet is to downgrade the [maven tailwindcss webjar dependency](https://mvnrepository.com/artifact/org.webjars.npm/tailwindcss). I believe the version I gave in my example is working (version 3.2.1) - but I have not used it in awhile. Alternatively you could play around with difference versions and find the latest working version and post it back on this thread for others
